### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kingdorian**
Updated the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# Contextproject-BeNine
\ No newline at end of file
+# Contextproject-BeNine
+
+
+## Frontend
+Installing the frontend:
+1. Make sure NodeJS v4 is installed on your system.
+2. Open a terminal inside the frontend directory.
+2. Run "npm instal".
```
Comment (jochem725) [inline README.md]: This should be "npm install"

**Comment by Kingdorian** [comment]
Whoops you're right I'll fix it.

**Comment by egedebruin** [comment]
Is it correct that this should be merged in master?

**Comment by Kingdorian** [comment]
@egedebruin Yes because its only a readme update. However, I see the formatting of the list is not correct.

**Comment by jochem725** [comment]
There is an issue regarding dependencies on windows using the node install guide. I'll try to look into it as soon as possible so the readme can be correctly updated.

**Comment by jochem725** [comment]
Since the problem is now fixed, this can be merged.

**Pull request merged**